.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: CI Tests
+
+on: push
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
```

Comment:
Do they have the tags for the exact Ubuntu release? I'd assume that "latest" moves. *This* repo isn't sensitive to system requirements (e.g. GDAL), but others might be.